cmd/bosun/bosun.example.toml

```diff
@@ -0,0 +1,85 @@
+# Hostname will be used when links are created in templates (i.e. acknowledge links)
+Hostname = "bosun.example.com"
+
+# The HTTP IP and Port to Listen on. Default is ":8070"
+HTTPListen = ":8080"
```

Comment:
@baweaver the tests also reference this file, so some things I set some non defaults for test.